SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -141,9 +140,12 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 		return false;
 	}
 
+	//Init InterestFactory
+	SpatialInterestFactory = MakeUnique<SpatialGDK::InterestFactory>(ClassInfoManager);
+
 	if (!bInitAsClient)
 	{
-		InterestFactory::CreateClientCheckoutRadiusConstraint(ClassInfoManager);
+		SpatialInterestFactory->CreateClientCheckoutRadiusConstraint();
```

Comment:
It was just to differentiate it from the type name but if we don't care about that then we can go with just the simple InterestFactory.

Reply:
that actually makes a lot of sense. I think `SpatialInterestFactory` is fine then. Or maybe the inverse could work, with the instance called `InterestFactory` and the class called `SpatialInterestFactory`, but up to you

Reply:
Yeah I think the inverse might be the ticket!

Reply:
I will make this change but I won't rename the file just yet InterestFactory.h -> SpatialInterestFactory.h

Once it is approved I will rename to avoid comparison issues.